src/covid19sim/simulator.py

```diff
@@ -1222,7 +1222,8 @@ def at(self, location, city, duration):
                         h.contact_book.sent_messages_by_day[cur_day].append(h.cur_message(cur_day))
                         self.contact_book.sent_messages_by_day[cur_day].append(self.cur_message(cur_day))
 
-            t_overlap = min(self.leaving_time, getattr(h, "leaving_time", 60)) - max(self.start_time, getattr(h, "start_time", 60))
+            t_overlap = (min(self.leaving_time, getattr(h, "leaving_time", self.env.ts_initial+SECONDS_PER_HOUR*TICK_MINUTE)) -
```

Comment:
Why are we adding `ts_initial` here? Will it not result in a totally different return value as compared to in the previous implementation?

Reply:
I see! we are subtracting it anyway from the second term.

Reply:
@pg2455 Per commit `8056cbc56b9e2733871bfb8001a8c8a928dafc53`'s message:

_Translate Human's start_time/leaving_time from relative to absolute timestamps._

Because the value before was `getattr(h, "start_time", 60)` (i.e., 60 ticks relative to simulation start, which used to be defined as T=0), the new absolute timestamp must be offset by `ts_initial`. But that's just fine, since that offset gets subtracted anyways.